### DISCUSSION TITLE
Can I use getConfig? / How

### DISCUSSION THREAD

**Discussion by JiriBandura** [orestbida/cookieconsent#221] [Q&A] [answered]
Hi again,
I have another question.

Since my project portfolio is large and includes several multilingual projects, I need to reduce the data that is downloaded.
My idea was split a main script to 2 separate script. (1 - main with all configuration 2 - only language).

Is there any possible way to do it?

**Comment by orestbida** [comment]
@JiriBandura,

can you be more specific? 

Do you need to lazyload languages (load a specific language only if needed) or do you simply want to have 2 different files (loaded entirely)?

If it's the second, then yes, it can be easily done.

**Comment by JiriBandura** [comment]
Hi,
I need 2 files.

In the first file there will be a complete initialization, including settings. (force, model position etc...)
In the second file there will be ONLY language.

```
cc.run({
    languages: {
        'en': {
            consent_modal: {........
```

At the moment I have it set so that there is only an initialization in the first file. And in the second file I have settings (force, model position etc...), including language.
So, if I need to change the settings, I have to do it for all the 'second' files, instead of the change taking place in the first one.

**Comment by orestbida** [comment, marked as answer]
@JiriBandura,

this is fairly easy to achieve!

You need to have a `.js` file — for the sake of this example I'll name it `cc_languages.js` — with the following structure:
```javascript
window.cc_languages = {
    'en': {
        consent_modal: {
            title: ' We use cookies!',
            description: '...',
            // ...
        },
        settings_modal: {
            // ...
        }
    },
    'de': {
        // ...
    }
}
```
You will then need to load `cc_languages.js` file before running the plugin:
```javascript

// obtain plugin
var cc = initCookieConsent();

// load languages
cc.loadScript('cc_languages.js', function(){
    // run plugin
    cc.run({
        current_lang: 'en',
        // ...
    
        languages: window.cc_languages
    });
});

**Comment by JiriBandura** [comment]
@orestbida 
That's work like a charm. Thanks a lot.
Actually we save about 300ms timeblocking in LCP.
